assets/scss/_base.scss

```diff
@@ -902,6 +902,12 @@ section#cncf {
   margin: 0;
 }
 
+//Table Content
+.tab-content table{
```

Comment:
A little bit of `border-spacing ` would look better here (It's fine if you don't)

Reply:
Done